spec/lib/process_settings/testing/monitor_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'process_settings/testing/monitor'
+
+describe ProcessSettings::Testing::Monitor do
+  let(:settings_array) { [] }
+  let(:logger) { Logger.new('/dev/null') }
+  let(:test_monitor) { described_class.new(settings_array, logger: logger) }
+
+  # TODO: This is the perfect place to use SharedExamples in order to state that
+  #       this ProcessSettings::Testing::Monitor should behave like ProcessSettings::Monitor
```

Comment:
Still hoping to see an example of that feature in use!

Reply:
Added 😄

Reply:
Woo-hoo!